portfolio/src/main/webapp/script.js

```diff
@@ -46,18 +46,30 @@ async function getDataServletText() {
  * print on the page
  */
 async function getDataJson() {
-    const data_response = await fetch('/data');
+
+    var comment_number = document.getElementById('comment-number').value;
```

Comment:
Same here, comment_number should be renamed.

Also, use lowerCamelCase instead of underscore_separated for variable names